solutions/devsprint-riccardo-1/OnboardingChallengeKit/Sources/OnboardingChallengeKit/Screens/List/Cell/RepositoryCellView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  RepositoryCellView.swift
+//  OnboardingChallenge
+//
+//  Created by Mateus Nazário Coelho on 16/12/21.
+//
+
+import UIKit
+
+class RepositoryCellView: UITableViewCell {
+    // MARK: Outlets
+    private lazy var labelRepositoryName: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 18)
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var labelRepositoryOwnerName: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textColor = .gray
```

Comment:
It's better if we use `.systemGray`. The `.systemXXX` versions of the colors automatically support the dark mode. With the plain version, we should have to handle them manually.